app/views/middleware_server/show.html.haml

```diff
@@ -6,4 +6,6 @@
     var miq_after_onload = "miqAsyncAjax('#{url_for(:action => @ajax_action, :id => @record)}');"
 - else
   = render :partial => @showtype
-= render :partial => 'deploy'
```

Comment:
This has been moved to a couple of lines below, but do we actually want to keep this commented and disable the deploy functionality ?